### ISSUE TITLE
Duplicate Virus model causes unit test to fail

### ISSUE THREAD

**Issue by nicolaspayette** [NetLogo/models#34]
We have a unit test in https://github.com/NetLogo/NetLogo/blob/5.x/src/test/org/nlogo/workspace/TestLibrary.scala that checks if every model has a unique name. It fails because of the `Virus` model:

```
[info] TestLibrary:
[info] - every model has a unique name *** FAILED ***
[info]   Virus.nlogo (TestLibrary.scala:14)
```

As of 5.2RC4, this model appears in both the `Biology` folder and the `Alternative Visualizations` folder. The two versions are _almost_ identical: only the value of the `turtle-shape` chooser differs between the two.

We have other models that are duplicated in the library, but those are copied by `release.sh`, after the unit tests are run. We cannot do that for `Virus` because the two versions differ, however slightly.

I see four options:

1. Disable the unit test altogether. Not a good idea in my opinion.
2. Add a special case to the unit test. That's the easiest course of action, and the one I expect we'll take, though special cases like this make NetLogo harder to maintain over time.
3. Rename the version in the `Alternative Visualizations` folder to something else (though the name `Virus - Alternative Visualization` is already taken).
4. Get rid of the duplicate model (I know, I know, that's not really an option... ;-))

@UriCCL, @arthurhjorth, @frankduncan, care to weight in?

(I'm tagging this 5.2, but just to be clear: this is not holding us up for release.)

**Comment by arthurhjorth** [comment]
3 seems like the best choice to me. Giving it another name doesn't cost
anything, and has no downsides.

-- 
Arthur Hjorth
PhD Student, Learning Sciences
Center for Connected Learning <http://ccl.northwestern.edu/>
Northwestern University

e: arthur.hjorth@u.northwestern.edu
s: arthurhjorth
m: +1 (773) 943-0013

**Comment by nicolaspayette** [comment]
@UriCCL says:

>Either 2 or 3 works for me.
> If we were to do 3, we’d probably name it virus - circle visualization

3 it is. Fixed in cc16399.
